context/app/static/js/hooks/useSearchData.ts

```diff
@@ -191,9 +191,15 @@ export function useMultiSearchData<Documents, Aggs>(
   const requestInits = useRequestInits({ queries, useDefaultQuery });
   const { elasticsearchEndpoint } = useAppContext();
 
+  const inlineFetcher = () =>
+    fetcher({
+      urls: [elasticsearchEndpoint],
+      requestInits,
+    }) as Promise<SearchResponseBody<Documents, Aggs>[]>;
+
```

Comment:
I think if we provide the function directly in the `useSWR` hook call we can avoid needing to cast its return type, but I could be mistaken